### PR TITLE
v0 Design Fixes

### PR DESCRIPTION
**Pull request by lvreynoso**
### Description

Oooo pretty!!

Screenshot from 2021-02-26 11-31-51

#### Issue
[ch66711](https://app.clubhouse.io/genepi/stories/space/66711)

### Test plan

Load app, navigate around, look at the sparklies.

### CODE REVIEW

**Comment by phoenixAja** [comment]
BEAUTIFUL!! ✨ I'm going to pull now and give it a look, we should definitely showcase this in the demo today.

File: src/ts/src/common/api/index.tsx [hunk 1]
```diff
@@ -20,8 +20,8 @@ export const fetchUserData = async (): Promise<
 const SAMPLE_MAP = new Map<string, keyof Sample>([
     ["collection_date", "collectionDate"],
     ["collection_location", "collectionLocation"],
-    ["private_identifier", "privateID"],
-    ["public_identifier", "publicID"],
+    ["private_identifier", "privateId"],
+    ["public_identifier", "publicId"],
     ["upload_date", "uploadDate"],
```
Comment (phoenixAja) [inline src/ts/src/common/api/index.tsx:25]: do we need to add gisaid to this now too?

Comment (lvreynoso) [reply src/ts/src/common/api/index.tsx:25]: No, since the gisaid key is not different between the api and the TS type it will transfer automatically :)

**Review by phoenixAja — APPROVED** [review]
design looks good on firefox! The only thing is if we should add the GISAID field now that @ttung added that functionality to the sample data view?

**Comment by ttung** [comment]
> design looks good on firefox! The only thing is if we should add the GISAID field now that @ttung added that functionality to the sample data view?

#138!

**Comment by lvreynoso** [comment]
gisaid field is there :) the `jsonToType<T>` function uses the mapping, but any key in the input object not in the mapping just gets directly transferred. It's already in the `Sample` type definition and the table def hits that key.

**Pull request merged**